src/views/DissolutionFirm/DissolutionFirm.vue

```diff
@@ -596,13 +596,6 @@ h2::before {
   }
 }
 
-#dissolution-date-section .v-card {
-  // remove extra space taken by error message
-  :deep(.v-text-field__details) {
-    margin-bottom: -8px !important;
-  }
-}
-
```

Comment:
This is no longer needed because the DatePicker now uses `hide-details="auto"`.

The other tweak is to have even whitespae above and below.